shell/pom.xml

```diff
@@ -75,6 +75,16 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-master</artifactId>
```

Comment:
Alluxio client jar will be distributed to many different applications. For example, it may need to distribute to a hadoop cluster with hundreds node. We want to keep client jar light weight. Not sure is it acceptable to make alluxio client jar depends on the core-server module. Ideally, we should have a separate alluxio-examples jar (now we include many tools and tests in the client jar which is kind of strange).

Reply:
Moved journal-tool to `core-server-master`.